plugins/jobs/src/js/JobsMenuClient.js

```diff
@@ -0,0 +1,31 @@
+import "rxjs/add/operator/combineLatest";
+import "rxjs/add/operator/map";
+
+import { makeExecutableSchema } from "graphql-tools/dist/index";
+import { runJob } from "#SRC/js/events/MetronomeClient";
+
+export const typeDefs = `
+  type Job {
+      id: String,
+      jobId: String,
+      status: String,
+      createdAt: String
+  }
```

Comment:
If we are only using the jobId right now, that would be the only thing I would put on this type.